src/components/Publish/Actions/index.tsx

```diff
@@ -78,6 +81,15 @@ export default function Actions({
             <Button type="submit" style="primary" onClick={handleActivation}>
               Connect Wallet
             </Button>
+          ) : !isSupportedOceanNetwork ? (
+            <Tooltip
+              content={<UnsuportedNetwork />}
+              className={styles.errorTooltip}
+            >
```

Comment:
> I don't think there's a need to introduce a specific unsupported networks tooltip.

I'm still using the same `Tooltip` component from `'@shared/atoms/Tooltip'` but I'm passing in a new component as the content: